views/package_version/_header.ejs

```diff
@@ -1,13 +1,29 @@
 <header class="package-header">
-  <div class="container">
+  <div class="package--navigation">
+    <div class="container">
+      <nav>
+        <ul>
+          <li><a href="<%= path %>#readme" class="btn">Readme</a></li>
```

Comment:
Why these tabs? This wasn't what we agreed about I think? Also, they don't seem to work entirely. Let's call about this.